opta/commands/force_unlock.py

```diff
@@ -28,23 +28,66 @@ def force_unlock(config: str, env: Optional[str]) -> None:
     amplitude_client.send_event(amplitude_client.FORCE_UNLOCK_EVENT)
     layer = Layer.load_from_yaml(config, env)
     layer.verify_cloud_credentials()
+    modules = Terraform.get_existing_modules(layer)
+    layer.modules = [x for x in layer.modules if x.name in modules]
     gen_all(layer)
 
-    Terraform.init(layer=layer)
+    if Terraform.download_state(layer):
+        tf_lock_exists, _ = Terraform.tf_lock_details(layer)
+        if tf_lock_exists:
+            Terraform.init(layer=layer)
+            click.confirm(
+                "This will remove the lock on the remote state."
+                "\nPlease make sure that no other instance of opta command is running on this file."
+                "\nDo you still want to proceed?",
+                abort=True,
+            )
+            tf_flags.append("-force")
+            Terraform.force_unlock(layer, *tf_flags)
 
-    click.confirm(
-        "This will remove the lock on the remote state."
-        "\n\tPlease make sure that no other instance of opta command is running on this file."
-        "\n\tDo you still want to proceed?",
-        abort=True,
-    )
+        if "k8scluster" in modules:
+            configure_kubectl(layer)
```

Comment:
I think there is a case where k8scluster is in modules but has not successfully run yet. so this would fail..

Reply:
Maybe a better check would be if tf outputs have the k8s cluster url?

Reply:
Here we are checking if the whole of the k8scluster module is created successfully, and that only appears once a single terraform apply succeeds.